pkg/providers/builtin/spanner/definition.go

```diff
@@ -70,7 +70,7 @@ func ServiceDefinition() *broker.ServiceDefinition {
 				FieldName: "name",
 				Type:      broker.JsonTypeString,
 				Details:   "A unique identifier for the instance, which cannot be changed after the instance is created.",
-				Default:   "pcf-sb-${counter.next()}-${time.nano()}",
+				Default:   "gsb-${counter.next()}-${time.nano()}",
```

Comment:
I just noticed that this uses hyphens where storage/definition.go uses underscores